lib/properties/torque.dart

```diff
@@ -42,6 +44,10 @@ class Torque extends DoubleProperty<TORQUE> {
               coefficientProduct: 1.35581794902490555,
               name: TORQUE.poundForceFeet,
             ),
+            ConversionNode(
+              coefficientProduct: 0.1129848290854088,
+              name: TORQUE.poundForceInch,
+            ),
```

Comment:
I suggest to implement it as 1/12 of a pound force feet

Reply:
to be honest I am not familiar with the laguage, do you have a suggestion on how to reference the value or should I just write `1.35581794902490555/12`

Reply:
Add the `ConversionNode` as a List to the `leafNode` property of the pound force feet `conversionNode`